src/renderer/object/texture_region.rs

```diff
@@ -0,0 +1,36 @@
+#[derive(Clone, Copy, Debug, Default)]
+pub struct TextureRegion {
```

Comment:
Maybe call it TextureTransform to be consistent with the [gltf standard](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_texture_transform)

Reply:
Done and done